Cargo.toml

```diff
@@ -153,3 +154,15 @@ debug = false
 strip = "debuginfo"
 [profile.dev.package."*"]
 opt-level = 3
+
+[patch."https://github.com/EspressoSystems/HotShot.git"]
```

Comment:
Can we remove the `patch` before merging? Just actually do the HotShot update in `dependencies`?

Reply:
I can try yeah

Reply:
I'm getting compile errors 😢

Reply:
Oh right because without the `patch` we'd have to go update the HotShot version in all the dependencies

Reply:
I suppose we can merge with the patch to unblock staging testing for the next release, but we should quickly go back and make this an official hotshot update

Reply:
Yeah it is already on HS `main`, but I have to update the weekly release